controllers/ibu_controller.go

```diff
@@ -219,14 +225,34 @@ func (r *ImageBasedUpgradeReconciler) handleAbortOrFinalize(ctx context.Context,
 	return
 }
 
-func isRollbackAllowed(ibu *lcav1alpha1.ImageBasedUpgrade) bool {
+var isAfterPivot = func(stateroot string) (bool, error) {
```

Comment:
Suggest renaming to something like runningUpgradedRelease, with no parameters. There should be no need to pass in the stateroot here, as it could use the ```ibu.Spec.SeedImageRef.Version``` internally. It's either running that release or it's not.
Alternatively, if you wanted to be able to pass in different stateroots for some reason, rename to ```isStaterootBooted```, since that's really what this is saying here.

Reply:
Why the "var" syntax here? This function would be a useful utility function elsewhere, for the Upgrade handler for example. I'm not sure what the purpose of the var here is